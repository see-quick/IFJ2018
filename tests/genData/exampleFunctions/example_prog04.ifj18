# Program 4: count square area


square_side = 0
area = 0

   print ("\nEnter the whole side of square\x20\n")
   square_side = inputi
   #calculation of the area
   area = square_side * square_side

   print("\nArea of the square\x20", area, "\n")
