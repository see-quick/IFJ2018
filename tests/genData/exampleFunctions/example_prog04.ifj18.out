
Enter the whole side of square 
5
5
Area of the square 25

